src/components/SnowFlakes.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
+import { useResponsiveValue } from '../hooks/useResponsiveValue';
+import { Snowfall, Snowflake } from 'react-snowflakes';
+import '../styles/SnowFlakes.css';
+
+const SnowFlakes = () => {
+    // TODO: rewrite component without library
+    // TODO: fix snowflakes colors
+
+    return (
+        <div className="snowflakes">
+            <Snowfall count={ useResponsiveValue(100, 10, 900) }
+                style={{
+                    position: 'relative',
+                    width: '100vw',
+                    height: '100vh'
+                }}
+                snowflakeFactory={index => {
+                    const size = 1; // 50 is the number of snowflakes.
+                    const w = 5 + 10 * size + 'px';
+                    return (
+                        <Snowflake speed={.5 + size * 2}
+                            xSpeedPrc={.3 * size}
```

Comment:
Please add comments to each value here (that's not clear) so you understand what it does later, and why you've put those numbers there.